src/_pytest/terminal.py

```diff
@@ -255,6 +251,22 @@ def _determine_show_progress_info(self):
             return False
         return self.config.getini("console_output_style") in ("progress", "count")
 
+    @property
+    def verbosity(self):
+        return self.config.option.verbose
+
+    @property
+    def showheader(self):
+        return self.verbosity >= 0
+
+    @property
+    def showfspath(self):
+        return self.verbosity >= 0
+
+    @property
+    def showlongtestinfo(self):
+        return self.verbosity > 0
```

Comment:
Same as above

Reply:
Agree, let's keep it, it is not like it is a huge burden anyway. 👍